src/routers/confirmCompanyRouter.ts

```diff
@@ -11,9 +13,10 @@ router.get("/", handleExceptions(async (req: Request, res: Response, _next: Next
 }));
 
 router.post("/", handleExceptions(async (req: Request, res: Response, _next: NextFunction) => {
-    const handler = new PscTypeHandler();
-    const { templatePath, viewData } = handler.execute(req, res);
-    res.render(templatePath, viewData);
+    // const handler = new PscTypeHandler();
+    // const { templatePath, viewData } = handler.executeGet(req, res);
+    // res.render(templatePath, viewData);
```

Comment:
remove the commented out lines, won't be needed

Reply:
Done